docs_src/content/docs/40-components.md

```diff
@@ -514,10 +519,24 @@ As opposed to `<tabView>`:
 
 ```html
 <segmentedBar items="{listOfItems}" selectedIndex="0"
-    on:selectedIndexChange="{onSelectedIndexChange}" />
+              on:selectedIndexChange="{onSelectedIndexChange}" />
+```
+
+`<segmentedBar>` can be populated with `{each}` block.
```

Comment:
Just to show that is more easy to use the `each` block. Since `items` prop is an Array of `<segmentedBarItem>`